ext/strings.go

```diff
@@ -1084,6 +1090,43 @@ func makeMatcher(locale string) (language.Matcher, error) {
 	return language.NewMatcher(tags), nil
 }
 
+// quote implements a string quoting function. The string will be wrapped in
+// double quotes, and all valid CEL escape sequences will be escaped to show up
+// literally if printed. If the input is not valid UTF-8, quote will return with
+// an error.
+func quote(s string) (string, error) {
+	var quotedStrBuilder strings.Builder
+	if !utf8.ValidString(s) {
```

Comment:
After thinking it over a bit more I'm not so sure this is the right approach. Golang [replaces any invalid runes when quoting](https://go.dev/src/strconv/quote.go), that would probably be better.

Reply:
(marking as resolved, invalid UTF-8 characters are now replaced)